package.json

```diff
@@ -107,5 +107,8 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "resolutions": {
+    "write-file-atomic": "2.4.1"
```

Comment:
Any chance you can elaborate on this change? :)

Reply:
Oh right! Sorry! Jest has a known issue that causes a race condition on Windows and breaks some tests. This is the recommended fix in the issue that tracks this bug (https://github.com/facebook/jest/issues/4444)